### PR TITLE
Add working tests

### PR DESCRIPTION
**Pull request by soberstadt**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 language: node_js
 node_js: "12"
 
-script: yarn lint
```
Comment (OzzieOrca) [inline .travis.yml:4]: Do you also want to run lint without the `--fix` flag?

File: test/bot.test.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        .reply(200, { content: encodedConfig })
+
+      // allow test to get PR details
+      nock('https://api.github.com')
+        .get('/repos/soberstadt/test-merge-repo/pulls/2')
+        .reply(200, { head: { ref: 'asdf1234' } })
+
+      mergeMock = nock('https://api.github.com')
+        .post('/repos/soberstadt/test-merge-repo/merges', (body) => {
+          console.log(body)
```
Comment (OzzieOrca) [inline test/bot.test.js:55]:
```suggestion
```

File: test/bot.test.js [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+          expect(body).toMatchObject(mergeBody)
```
Comment (OzzieOrca) [inline test/bot.test.js:56]: Do you want expectations in the nock callback? Usually it's better to do them inside an `it` block.

File: .travis.yml (hunk 1 above)
Comment (soberstadt) [reply .travis.yml:4]: My understanding is that `yarn test` is the same thing as `npm run test`, which will run `jest && standard` returning an error if either are unsuccessful.

File: test/bot.test.js (hunk 3 above)
Comment (soberstadt) [reply test/bot.test.js:55]: I don't know if nock supports evaluating the requests after they have run. I could move the mocks with `expect` in them into the `it` blocks, but it seemed like this was cleaner. But I could go either way on it. Do you have a different network mocker that you prefer?

File: .travis.yml (hunk 1 above)
Comment (OzzieOrca) [reply .travis.yml:4]: Oh I missed that that script was doing 2 things...

File: test/bot.test.js (hunk 3 above)
Comment (OzzieOrca) [reply test/bot.test.js:55]: 🤷‍♂️ This is fine.

**Pull request merged**